tools/rpc-kast.sh

```diff
@@ -8,7 +8,7 @@ profile=false
 verbose=false
 
 run_pretty_json_request() {
-    kore_json="$(jq .term $1)"
+    kore_json="$(jj -i $1 params.state.term)"
```

Comment:
possibly rever/refactor into a separate pr? jq didnt work because of its input limitation